internal/cmd/base/servers.go

```diff
@@ -692,6 +707,10 @@ func (b *Server) SetupWorkerPublicAddress(conf *config.Config, flagValue string)
 	return nil
 }
 
+func looksLikeTemplate(s string) bool {
+	return strings.HasPrefix(s, "{{") && strings.HasSuffix(s, "}}")
+}
+
```

Comment:
This is not my favourite thing in the world, but in my head the alternative is actually trying and checking the error, and even that isn't 100% reliable because `ParseSingleIPTemplate` doesn't error in cases like, for example, `"bla"` is input.

Do we have any different ideas here, or do we think this is OK?